frb/scripts/pz_dm.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+"""
+Estimate p(z|DM) for an assumed location on the sky and DM_FRB
+"""
+from IPython import embed
+
+def parser(options=None):
+    import argparse
+    # Parse
+    parser = argparse.ArgumentParser(description='Script to print a summary of an FRB to the screen [v1.0]')
+    parser.add_argument("coord", type=str, help="Coordinates, e.g. J081240.7+320809 or 122.223,-23.2322 or 07:45:00.47,34:17:31.1 or FRB name (FRB180924)")
+    parser.add_argument("DM_FRB", type=float, help="FRB DM (pc/cm^3)")
+    parser.add_argument("--dm_hostmw", type=float, default=100., help="Assumed DM contribution from MW and Host")
+    parser.add_argument("--cl", type=tuple, default=(2.5,97.5), 
```

Comment:
Maybe allow for a scalar input here to reduce user error. i.e. if someone wants 95% c.l around the median, they should not need to input two numbers.

Reply:
I prefer to keep it explicit, i.e. input 2 numbers.  Thx!